File: javascript/7_kyu/cw_shortestword.js
function findShort(s){
  var array = s.split(" ");
  var shortest = array[0].length;
   for (var i = 0; i < array.length; i++) {
     if (array[i].length <= shortest) {
     shortest = array[i].length
     }
   }
   return shortest
}

// https://www.codewars.com/kata/57cebe1dc6fdc20c57000ac9

File: ruby/7_kyu/cw_shortestword.rb
def find_short(s)

  word_lengths = []
  words = s.split(" ")
  
  words.each do |word|
    word_lengths << word.length
  end
  
  l = word_lengths.min
 
  return l # l: length of the shortest word
end

# https://www.codewars.com/kata/57cebe1dc6fdc20c57000ac9